kythe/typescript/test.ts

```diff
@@ -128,9 +128,38 @@ async function testIndexer(args: string[]) {
   return 0;
 }
 
+async function testPlugin() {
+  const plugin: indexer.Plugin = {
+    name: 'TestPlugin',
+    index(
+        pathToVName: (path: string) => indexer.VName, paths: string[],
+        program: ts.Program, emit?: (obj: {}) => void) {
+      for (const testPath of paths) {
+        const relPath = path.relative(
+                                program.getCompilerOptions().rootDir!,
+                                program.getSourceFile(testPath)!.fileName)
+                            .replace(/\.(d\.)?ts$/, '');
+
+        const pluginMod = {
+          ...pathToVName(relPath),
+          signature: 'plugin-module',
+          language: 'plugin-language',
+        };
+        emit!({
+          source: pluginMod,
+          fact_name: '/kythe/node/kind',
```

Comment:
Just for ensure it's clear we're not "inheriting" the fact from the indexer proper, use some other test fact and value, and verify that.